vars/collectAdditionalInfrastructureVariablesFor.groovy

```diff
@@ -1,15 +1,9 @@
 #!groovy
 import uk.gov.hmcts.contino.azure.KeyVault
-import uk.gov.hmcts.contino.azure.ProductVaultEntries
 
 def call(subscription, product, environment) {
   KeyVault keyVault = new KeyVault(this, subscription, "${product}-${environment}")
   def environmentVariables = []
 
-  def appInsightsInstrumentationKey = keyVault.find(ProductVaultEntries.APP_INSIGHTS_INSTRUMENTATION_KEY)
```

Comment:
I mean delete this whole file `vars/collectAdditionalInfrastructureVariablesFor.groovy` and remove anything that is calling it